src/core/sm/layermanager/layermanager.cpp

```diff
@@ -490,13 +490,13 @@ Error LayerManager::InstallLayer(const LayerInfo& layer)
     Error                               err = ErrorEnum::eNone;
     UniquePtr<spaceallocator::SpaceItf> downloadSpace;
     UniquePtr<spaceallocator::SpaceItf> unpackedSpace;
-    StaticString<cFilePathLen>          archivePath;
-    StaticString<cFilePathLen>          storeLayerPath;
+    auto                                archivePath    = MakeUnique<StaticString<cFilePathLen>>(&mAllocator);
```

Comment:
Now InstallLayer and InstallService functions look different. I guess `fs::JoinPath` also consumes stack. Probably we need to increase stack usage limit to 8k. Let's discuss.

Reply:
Done